src/Module/Xrd.php

```diff
@@ -272,57 +272,64 @@ private function printXML(string $alias, array $owner, array $avatar)
 					]
 				],
 				'4:link' => [
+					'@attributes' => [
+						'rel'  => 'self',
+						'type' => 'application/activity+json',
```

Comment:
Does `self` always points to the JSON representation or should it return the XML content type here, since we are delivering the XML here?

Reply:
To my knowledge & from testing, requesting `application/xml` or `application/activity+xml` here just returns HTML.

Reply:
What does the specifications say?